api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -48,4 +50,14 @@ message QuicProtocolOptions {
   // bound by 6000, regardless of this field or how many connections there are.
   google.protobuf.UInt32Value packets_to_read_to_connection_count_ratio = 5
       [(validate.rules).uint32 = {gte: 1}];
+
+  // Configure which implementation of quic::QuicCryptoClientStreamBase to be used for this listener.
```

Comment:
could you surround `quic::QuicCryptoClientStreamBase` with ` `` ` as its a letiral